main.js

```diff
@@ -74,7 +74,7 @@ function renderTradeDialog(townName, town, products, pricesInTown) {
   container.append(nameLabel);
   
   const table = $('<table>');
-  const header = $('<tr><th></th><th>Available</th><th>Price</th><th>Sell</th><th>Buy</th></tr>');
+  const header = $('<tr><th></th><th>Available</th><th>Price</th><th>Sell</th><th>Sell</th><th>Buy</th><th>Buy</th></tr>');
```

Comment:
I think it would look good if there was only 1 Sell and Buy header, as such:
```
const header = $('<tr><th></th><th>Available</th><th>Price</th><th></th><th>Sell</th><th>Buy</th><th></th></tr>');
```